samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -1012,7 +1012,7 @@ static int link_shell_dnsaddr(const struct shell *shell, size_t argc, char **arg
 static int link_shell_edrx(const struct shell *shell, size_t argc, char **argv)
 {
 	int ret = 0;
-	enum link_shell_common_options common_option = LINK_COMMON_NONE;
+	enum link_shell_common_options operation = LINK_COMMON_NONE;
```

Comment:
This is intentionally still using the enum kind of approach while other functions use a boolean for each common option.

Reply:
Decided to use enum